src/main/java/com/stage/users/Controllers/UserController.java

```diff
@@ -0,0 +1,83 @@
+package com.stage.users.Controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.stage.users.Iservice.UserIservice;
+import com.stage.users.Model.User;
+
+
+@RestController
+@RequestMapping("users")
+public class UserController {
+	
+	@Autowired
+	UserIservice userservice;
+	
+	
+	@PostMapping("/adduser")
+	public ResponseEntity AddUser(@RequestBody User user) {
+		User usersave = null ; 
+	try {
+		usersave = userservice.addUser(user);;
+	} catch (Exception ex) {
+		ex.printStackTrace();
+        return  ResponseEntity.badRequest().body(ex.getMessage());
+	}
+		return ResponseEntity.status(HttpStatus.CREATED).body(usersave);
+	}
+
+
+		@GetMapping("/getallusers")
```

Comment:
.....users/getallusers as uri here ? 
better not : better set it as @GetMapping simply because it will have the users as URI, and that's convenient